test/defs/test_defs_au.rb

```diff
@@ -11,15 +11,15 @@ def test_au
 
     assert_equal "ANZAC Day", (Holidays.on(Date.civil(2020, 4, 25), [:au_qld])[0] || {})[:name]
 
-    assert_equal "Additional public holiday for ANZAC Day", (Holidays.on(Date.civil(2021, 4, 26), [:au_wa])[0] || {})[:name]
+    assert_equal "Additional public holiday for ANZAC Day", (Holidays.on(Date.civil(2021, 4, 27), [:au_wa])[0] || {})[:name]
```

Comment:
I'm not sure why `rake generate` led to changes in this file. Have we not been updating these files judiciously? Naughty.